.github/workflows/build_and_push_docker_images.yaml

```diff
@@ -57,38 +58,33 @@ jobs:
             echo 'is_latest=false'
             is_latest=false
           fi
-          if git describe --tags --match "[v|e]*" --exact; then
+          # resolve profile
+          if git describe --tags --match "v*" --exact; then
+            echo "This is an exact git tag, will publish images"
+            is_exact='true'
+            PROFILE=emqx
+          elif git describe --tags --match "e*" --exact; then
             echo "This is an exact git tag, will publish images"
             is_exact='true'
+            PROFILE=emqx-enterprise
           else
             echo "This is NOT an exact git tag, will not publish images"
             is_exact='false'
           fi
-          case $tag in
-            refs/tags/v*)
-              PROFILE='emqx'
+
+          case "${PROFILE:-$INPUTS_PROFILE}" in
```

Comment:
Q: shouldn't it be the other way around?  Like `${INPUTS_PROFILE:-PROFILE}`?

That way, if we use a workflow dispatch, we can force the profile we specified.

Reply:
If we trigger this manually, then it most likely going to be on an existing tag. Can't think of a scenario when we'd need to override profile.